static/src/javascripts/projects/commercial/commercial-metrics.ts

```diff
@@ -7,24 +10,25 @@ const { browserId } = window.guardian.config.ophan;
 const { adBlockers } = window.guardian;
 
 let logged = false;
-const sendMetrics = (): void => {
+const initMetrics = (): void => {
 	if (logged) return;
 	const args: [string, string | undefined, boolean, boolean?] =
 		adBlockers.active === undefined
 			? [pageViewId, browserId, isDev]
 			: [pageViewId, browserId, isDev, adBlockers.active];
 
-	logged = sendCommercialMetrics(...args);
+	logged = initCommercialMetrics(...args);
 };
 
 const init = (): Promise<void> => {
-	if (!window.guardian.config.switches.commercialMetrics)
+	if (!window.guardian.config.switches.commercialMetrics) {
 		return Promise.resolve();
+	}
 
-	const userIsInSamplingGroup = Math.random() <= 0.01;
+	initMetrics();
 
-	if (isDev || shouldCaptureMetrics() || userIsInSamplingGroup) {
-		document.addEventListener('visibilitychange', sendMetrics);
+	if (shouldCaptureMetrics()) {
+		bypassCommercialMetricsSampling();
```

Comment:
MInor subjective thing so totally up to you.

To me this reads as `shouldCapureMetrics` -> `bypassMetrics` i.e. don't capture.

I guess it actually means `bypassSampling` i.e. don't sample.

```suggestion
	if (shouldCaptureMetrics()) {
		switchOffSampling();
```

Reply:
This is defined in `commercial-core`, alas.

You might want to bring this renaming upstream if it's clearer?

Reply:
Yes, that's a good point. It's probably worth making the change upstream now. The alternative use an alias and do a card for the change later but then it would require two further PRs for Frontend and DCR as well.